ledfx/devices/launchpad.py

```diff
@@ -0,0 +1,202 @@
+import logging
+
+import voluptuous as vol
+from numpy import zeros
+
+import ledfx.devices.launchpad_lib as launchpad
+from ledfx.devices import MidiDevice
+
+# import timeit
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def dump_methods(instance):
```

Comment:
I found this to be a REALLY handy util for debugging supported methods when relational browsing could not handle things.